basic/src/test/java/org/jboss/hal/testsuite/test/runtime/server/InfiniteServlet.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.hal.testsuite.test.runtime.server;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@WebServlet("/infinity")
+public class InfiniteServlet extends HttpServlet {
```

Comment:
Personally I would prefer the name of servlet to match what the servlet does and not naming it just based on test purpose.

From its name I would expect that it will do something for infinity, which doesn't correspond to reality.